src/v2/view-builder/internals/BaseOktaVerifyChallengeView.js

```diff
@@ -157,18 +160,40 @@ const Body = BaseFormWithPolling.extend({
     };
 
     let probeChain = Promise.resolve();
+    // If https domain exists, do https domain probe first
+    // This only applies to MacOS for now
+    if (authenticatorHttpsDomainUrl) {
+      // if https domain are included, max number of ports to be probed should be doubled
+      Logger.info('httpsDomain enabled, will probe and challenge https first');
+      maxNumberOfPorts += maxNumberOfPorts;
+      ports.forEach(port => {
```

Comment:
move this probing logic into a function? looks similar with the one used for regular domain.